marshaler.go

```diff
@@ -632,9 +636,11 @@ func (enc *Encoder) encodeTableInline(b []byte, ctx encoderCtx, t table) ([]byte
 	return b, nil
 }
 
-var errNilInterface = errors.New("nil interface not supported")
-
-var textMarshalerType = reflect.TypeOf(new(encoding.TextMarshaler)).Elem()
+var (
+	errNilInterface = errors.New("nil interface not supported")
+	//nolint:gochecknoglobals
```

Comment:
I think this linter can be disabled.

Reply:
I hope this won't come bite later! 
commented the linter and removed traces of it